packages/icons/package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "@cockroachlabs/icons",
+  "version": "0.1.0-prerelease.0",
+  "description": "Icons for Cockroach UI exported as React Components",
+  "files": [
+    "dist/main.js",
+    "dist/main.d.ts"
+  ],
+  "scripts": {
+    "build": "webpack --mode development",
```

Comment:
Does the mode matter?

Reply:
This does matter, in that I wasn't setting a mode in the webpack config. To simplify matters, I'm going to set the mode to `none` in the config and take out the switch.